pkg/assembler/backends/inmem/backend.go

```diff
@@ -104,7 +104,7 @@ func (n *noKnownVuln) Neighbors() []uint32 {
 }
 
 func (n *noKnownVuln) BuildModelNode(c *demoClient) (model.Node, error) {
-	return model.NoVuln{nodeID(n.id)}, nil
+	return model.NoVuln{ID: nodeID(n.id)}, nil
```

Comment:
```suggestion
	return &model.NoVuln{ID: nodeID(n.id)}, nil
```

(Parth has the similar change in his PR, to fix type casting)

Reply:
This PR is minor, so lets merge after #683 and I'll rebase

Reply:
rebased and resolved conflict.